tuned/daemon/daemon.py

```diff
@@ -122,6 +122,16 @@ def _notify_profile_changed(self, profile_names, result, errstr):
 		return errstr
 
 	def _full_rollback_required(self):
+                # if we are in chroot, we cannot use systemctl
+                retcode_root, out_root = self._cmd.execute(["stat", "-c", "%d:%i", "/"])
```

Comment:
Please use `os.stat()` (https://docs.python.org/2/library/os.html#files-and-directories). It is highly desirable to execute external programs as little as possible. And there's no need to execute the `stat` program here, as far as I can tell.

Also, notice that you're indenting with spaces, wheres the rest of the file is indented with tabs. It won't even bytecompile with python3.

Reply:
Ok so i can change the indentation, sorry that it is just related to my editor.
Regarding the stat, can you help me see how to get the right info from python os.stat? so i am basing the condition on device number and inodes, that i get with -c %d:%i flag is bash stat. I looked at the result of os.stat() but i cold not find the same results that i needed

Reply:
I actually could see there are other ways (the check for inode doesn't seem to pass in all cases). Like, for example: https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker
Do you prefer that alternative for checking if we are in a container, or do you hae some other suggestions that could be better?